contributing/core-contributor-getting-started.md

```diff
@@ -0,0 +1,27 @@
+# Getting started with being a core contributor to MDC iOS
+
+Checklist:
+
+- [ ] [Install git-clang-format](#git-clang-format).
+- [ ] Install https://github.com/material-foundation/github-squash-and-merge-pr-descriptions. This ensures that your pull request's merge commit matches your PR description.
```

Comment:
I haven't installed this and I have no problem writing commit messages. In some cases, the summary/description isn't updated and so I've gone and fixed it when I had to manually copy-paste. 

I think these are better listed as optionals/helpful tools. Having them as a "checklist" makes them feel required.